lib/components/narrative/default/itinerary-description.tsx

```diff
@@ -68,7 +65,7 @@ export function ItineraryDescription({ itinerary }: Props): JSX.Element {
   let transitMode
   itinerary.legs.forEach((leg) => {
     const { duration, mode, rentedBike, rentedVehicle } = leg
-    if (isTransit(mode) && duration > primaryTransitDuration) {
+    if (isTransitLeg(leg) && duration > primaryTransitDuration) {
```

Comment:
Maybe make this check a variable? It's used twice, here and 35

Reply:
I thought about that too but decided against it because it's a simple check anyway